### ISSUE TITLE
Improve support for multiple/conditional classes

### ISSUE THREAD

**Issue by yateam** [tipsy/j2html#40]
Hello, it seems like multiple css classes don;t work.

e.g.

I want something like this:
`<div class="class1 class2">`

to produce it I use :
`div().withClass("class1").withClass("class2")`

and this results in `<div class="class2">`

I assume, I need to use `div().withClass("class1 class2")`, right?

But it's not really convenient and not intuitive.

Consider also the following case: 
```
div().
.withClass("class1")
.withCondClass(isActive, "class2-active")
```
This also won't work but now there is no obvious way to write it correctly.
I guess I will have to do something like this:
```
String activeClass = isActive ? "class2-active" :  "";
div().withClass("class1 " + activeClass);
```

Which could also be improved should `withClass` work in additive way.

Thanks!

**Comment by tipsy** [comment]
When designing the API, I thought that "with" is closer to "set" than to "add", so the intended use is indeed `withClass("class1 class2")` as you guessed. Maybe we can solve it differently by introducing some sort of class-builder 🤔

**Comment by yateam** [comment]
I would not add any sort of class builders - that way you make your simple library unnecessary complicated IMHO. Maybe just add (rename existing) a method `addClass` and `addClassCond` - this way you give a user an idea what is going on when they use these methods...

Btw, I would also do something similar with `style` property. I see there is no specialized method for setting a style...

**Comment by tipsy** [comment]
With class-builders I just mean 
`div().withClass(class("class1").condClass("class2", 1 == 1))`.
I want to avoid the word (an operation) `add`, as it's so imperative.
About `style`: A lot of attributes didn't get convenience methods, so for style you're suppoed to use `withAttr("style", "color:red;")`

**Comment by yateam** [comment]
Works as well too!

as for styles - I think they are closer to classes - so they are not like other attributes. But definitely it's up to you to provide a special method for styles.

**Comment by tipsy** [comment]
When I designed the API I felt very strongly about not inlining styles, but I don't anymore. I can add a `withStyle()`,

**Comment by tipsy** [comment]
@yateam What do you think about

```
div().withClass(values().add("c1").condAdd("c2", true))
```

**Comment by yateam** [comment]
Yes, it's better!

**Comment by tipsy** [comment]
@yateam I've had this on my mind for a long time. The value builder turned out to be a bit too much. I have introduced a simple `addClass()` method that can be used instead of (or in addition to) `withClass()`. I'm not completely happy with the API, but it's the best I've been able to come up with. I'll create a PR.

**Comment by tipsy** [comment]
Please give feedback in #49. I dislike the way it breaks the fluidity of the API, but I also recognize that class is a special case.

**Comment by tipsy** [comment]
With #51 it would be possible to do `div().withClasses("c1", iff(1 == 1, "c2"))`

**Comment by tipsy** [comment]
I merged #51
